bom/pom.xml

```diff
@@ -69,7 +69,7 @@
       <version.groovy>2.4.8</version.groovy>
       <version.hibernate.javax.persistence>1.0.0.Final</version.hibernate.javax.persistence>
       <version.hibernate.core>5.2.7.Final</version.hibernate.core>
-      <version.hibernate.search>5.8.0.Beta3</version.hibernate.search>
+      <version.hibernate.search>5.8.0.Beta4</version.hibernate.search>
```

Comment:
Unfortunately the info in maven is not correct for everything :( We still have to work out the best way to correct that in an automated way, but until then it has to be done manually.